test/linter/test-versions.js

```diff
@@ -72,6 +72,48 @@ function removedAfterAdded(statement) {
   );
 }
 
+/**
+ * @param {VersionValue} xver
+ * @param {VersionValue} yver
+ */
+function compareVersionNumber(xver, yver) {
+  /**
+   * @param {VersionValue} version
+   */
+  function toArray(version) {
+    if (typeof version === 'string') {
+      return (version.startsWith('≤') ? version.slice(1) : version).split('.');
```

Comment:
I think we should use compare-versions to compare versions (with "≤" removed first).